starport/templates/app/stargate/config.yml

```
@@ -6,3 +6,9 @@ accounts:
 validator:
   name: alice
   staked: "100000000stake"
+faucet:
```

```suggestion
faucet:
  name: bob
  coins: ["5token"]
```

To make the default config smaller, let's omit coins max and instead of using Alice (the validator), let's use Bob.

---

removed by 0a4ec76 and ba94eda.